src/main/java/org/cloudfoundry/promregator/fetcher/TextFormat004Parser.java

```diff
@@ -130,9 +130,7 @@ private void parseMetric(String line) {
 			}
 			end = mValue.end();
 		}
-		
-		rest = rest.substring(end);
```

Comment:
@warlockza I get second thoughts here due to https://sonarcloud.io/project/issues?assignees=eaglerainbow%40github&createdAt=2019-12-01T22%3A59%3A09%2B0100&id=promregator&open=AW7DejBXmL6cMz3jBnyy

Either we set a value in line 116 or in 131. And they are usually not exactly the end of the string...

Reply:
Ah, perhaps you mean that nobody is using `rest` anymore then. That's because of the commented coding in line 139f.
Logically, instead of removing it, we most likely should also comment the `rest` and `end` above.

Reply:
see also #136